cmd/cstor-pool-mgmt/pool/pool.go

```diff
@@ -24,8 +24,27 @@ import (
 	"github.com/golang/glog"
 	apis "github.com/openebs/maya/pkg/apis/openebs.io/v1alpha1"
 	"github.com/openebs/maya/pkg/util"
+	"github.com/pkg/errors"
 )
 
+var (
+	poolTypeMap    map[string]string
```

Comment:
using these as global variables is not correct.

Reply:
This is being used at two places, once for validation and again for creating command attributes.